posh-git.psm1

```diff
@@ -11,26 +11,79 @@ if ($psv.Major -lt 3 -and !$NoVersionWarn) {
     "To suppress this warning, change your profile to include 'Import-Module posh-git -Args `$true'.")
 }
 
-Push-Location $psScriptRoot

```

Comment:
Are we just getting lucky that nobody using PS v2 has complained about this breaking anything?